### ISSUE TITLE
Allow editing changset comment

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/openstreetmap-website#3807]
It is very easy in various editors to submit a botched changeset comment.
https://github.com/MarcusWolschon/osmeditor4android/issues/1919
Therefore it would be great if one could be able to amend it after the fact.

**Comment by SomeoneElseOSM** [comment]
Everyone can of course add something to the changeset discussion that explains what they _should_ have said.

**Comment by tomhughes** [comment]
I believe the API allows that so it's up to editor's to expose that functionality. I think you can do it in JOSM for example.

Note that you can only do it while the changeset is open - once it is closed no more changes can be mode.

**Comment by tomhughes** [comment]
Yes you can do a PUT on the changeset to change it's tags while it's open.

We don't want to allow it after that because people will start consuming it and as changesets aren't versioned that will have no way to get updates or knows that anything has changed.
